PRIVACY.md

```diff
@@ -32,5 +32,20 @@ URL. On Chrome and Chromium, the `PRT SSO Cookie` is requested periodically
 with a generic URL. The returned token is injected into all http requests
 hitting the Entra ID login URL.
 
+### Note on required and optional host permissions
+
+We use the `WebRequest` (Firefox) or `declarativeNetRequest` (Chrome) API to
+inject the `PRT SSO Cookie` into requests targeting the login provider. To support
+this, we need the permission to access your data on `https://login.microsoftonline.com/`.
+This permission is (usually) requested at extension install time (required permission).
+
+For single-page applications (SPAs, like the Teams PWA) that perform automated token
+refreshes in the background, we further need the permission to access your data on
+the corresponding domains. To minimize the number of permissions we request, we provide users
+with the ability to grant these permissions on a case-by-case basis via the extension's UI or policy settings.
+Once granted, users can also revoke these permissions through the same interface.
+
```

Comment:
```suggestion
with the ability to grant these permissions on a case-by-case basis via the extension's UI or policy settings.
Granted permissions can also be revoked through the same interface.

```

I think this expresses the same idea slightly more clear.

With a start like "Once granted" I expect something that "I can finally do" but also "I want to do it right now". While it is technically also true that I can revoke permissions only after I granted them, it's usually not wanted to be done right after granting them.

Reply:
(github sucks: click on "unresolve", and your comment draft goes to /dev/null)
I'm still missing a description of what the user needs to do / can expect after installing the addon. I understand that I want to grant access to https://login.microsoftonline.com right from the beginning. But what's next? I enable also access to "all websites"? Or I rather wait until some app tries to re-login, fails, creates some entry into addon UI which I then can use to add individual website access?